scripts/deploy.js

```diff
@@ -1,18 +1,21 @@
-require("dotenv").config();
+require('dotenv').config()
 
-const hre = require("hardhat");
+const hre = require('hardhat')
 
 const main = async () => {
-  const podcastNFTContract = await hre.ethers.getContractFactory("PodCastNFT");
-  const contract = await podcastNFTContract.deploy([process.env.ADMIN_ADDRESS], false);
+  const podcastNFTContract = await hre.ethers.getContractFactory('PodCastNFT')
+  const contract = await podcastNFTContract.deploy(
+    process.env.TOKEN_ADDRESS,
+    process.env.ADMIN_ADDRESS
```

Comment:
[should] to avoid confusion, name should be FUND_ADDRESS. 
FUND_ADDRESS should be controlled by only multisig